packages/utils/src/groth16-proof.ts

```diff
@@ -0,0 +1,38 @@
+import { Groth16Proof } from "snarkjs"
+import { PackedProof } from "./types"
+
+/**
+ * Packs a proof into a format compatible with EddsaProof.
```

Comment:
I don't think we should mention other packages here. This should be agnostic.

Same below.

Reply:
Yes, I'm updating that part now.